airdrop-cli/package.json

```diff
@@ -19,9 +19,8 @@
     "cli:debug": "node dist/index.js debug",
     "cli:start": "node dist/index.js",
     "cli:tally": "node dist/index.js tally",
-    "cli:tally-multi-csv": "node dist/index.js tally-multi-csv",
-    "cli:tally-from": "node dist/index.js tally-from",
     "cli:single": "node dist/index.js single",
+    "cli:unspent": "node dist/index.js unspent",
     "cli:help": "node dist/index.js help",
```

Comment:
Why not always just use `tsx` why are you compiling and using node?

Reply:
Running ``tsx`` without having built it and the ``index.ts`` doesn't function, I'd need to create an IIFE for each command like I did with ``/invoke/index.ts`` and run each like a script.